src/main/scala/org/spot/SpotLDACWrapper.scala

```diff
@@ -1,4 +1,4 @@
-package org.opennetworkinsight
+package org.spot
```

Comment:
the incubating project "Apache Rya (incubating)" does not do it
so I guess there is flexibility